AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/MutationSync/OutgoingMutationQueue/ProcessMutationErrorFromCloudOperation.swift

```diff
@@ -92,6 +92,9 @@ class ProcessMutationErrorFromCloudOperation: AsynchronousOperation {
                 // TODO: dispatch Hub event
                 log.debug("Unauthorized mutation \(errorType)")
                 finish(result: .success(nil))
+            case .operationDisabled:
```

Comment:
On Android, I've only added logic to ignore `OperationDisabled` errors in the `SubscriptionProcessor`, not for mutations or sync.   What was thinking for adding it here as well?

Reply:
I believe an "OperationDisabled" error can be also thrown from a mutation resolver, so I thought it'd better handle it as we do for subscriptions

Reply:
It's not part of the current set of requirements, but I could imagine it is possible someone could want that in the future.  
 I don't feel strongly either way.  I didn't add it for Android, but I think it's fine to add if you want.